APIManager.js

```diff
@@ -0,0 +1,67 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {}
+    }
+
+    getUsers(){
+        const mainUser = {}
+        const friends = {friends: []}
```

Comment:
no need for `friends` to be an object, it could just be an array: `const friends = []`, and afterwards assign it to `this.data.friends = friends`